src/repository/users.rs

```diff
@@ -0,0 +1,31 @@
+use serde::Serialize;
+use sqlx::{prelude::FromRow, types::chrono};
+
+use super::Repository;
+
+#[derive(Debug, FromRow, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct User {
+    pub id: i64,
+    pub name: String,
+    pub traq_id: Option<String>,
+    pub github_id: Option<String>,
+    pub icon_url: String,
+    pub x_link: Option<String>,
+    pub github_link: Option<String>,
```

Comment:
多分WIPだと思うけど、post_problems, submut_problemsを追加すること
```suggestion
    pub github_link: Option<String>,
    //to be impremented
    //pub post_problems:
    //pub submut_problems:
```